### PR TITLE
Introduce a new, fast, build method

### PR DESCRIPTION
**Pull request by ktf**

### CODE REVIEW

**Comment by ktf** [comment]
@mpuccio could you review and eventually approve?

File: building/devel.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+This will fill your environment with all the requirements needed to properly build O2.
+
+You can checkout and build most of the AliceO2Group packages with:
+
+```bash
+git clone https://github.com/AliceO2Group/AliceO2 O2
+mkdir O2/objs
+cd O2/objs
+cmake .. -DBUILD_TEST_ROOT_MACROS=OFF
+make -j 20
```
Comment (mpuccio) [inline building/devel.md:33]: Maybe just replace `make` with `cmake --build . --target install -j8`. I put 8 as most of people (like me :( ) still don't have newer, better cpus.

**Review by mpuccio — CHANGES_REQUESTED** [review]
This is an excellent new feature, I would just change the small thing about make vs cmake --build and then we can merge

File: building/devel.md (hunk 1 above)
Comment (ktf) [reply building/devel.md]: Done.

**Review by mpuccio — APPROVED** [review]

**Comment by ktf** [comment]
Notice I cannot merge it...

**Comment by mpuccio** [comment]
Yes, as a matter of fact I cannot either because the test failed. Can you pull from my master branch? There is a fix there.

**Comment by ktf** [comment]
Done

**Review by mpuccio — APPROVED** [review]

**Pull request merged**